### ISSUE TITLE
[18.10] Active Directory authentification slow down webui

### ISSUE THREAD

**Issue by hdep** [centreon/centreon#6969]
Hello,

just installed 18.10 from ISO, configure AD authentification, Import my account into centreon.
Login to centreon is extremely slow it takes exactly 1min 27s

Please see the logs : 
[logs.txt](https://github.com/centreon/centreon/files/2580038/logs.txt)

I got the same behaviour when trying to export and reload pollers.

Any help appreciated.

**Comment by wackou72** [comment]
Hello,
mine is also slow, when connecting to the webinterface, it take arround 10 sec or more to connect with an AD account.
Generating the conf take arround 20 sec/30 sec.

**Comment by lpinsivy** [comment]
hi @hdep,

Your AD configuration is complexe? Do you use subgroups? In how many groups your user is linked?

**Comment by hdep** [comment]
Hi @lpinsivy 
yes my account is in 54 groups. I do'nt really know if there is group in groups.

**Comment by hdep** [comment]
Hi did you solve the issue in  #7057  ?

**Comment by hdep** [comment]
Same issue with 18.10.2

**Comment by lolokai** [comment]
Hello,
Up for me : we meat the same issue than @hdep

**Comment by joschi99** [comment]
Hello,
same issue with 18.10.3. Login using AD takes very long time.

**Comment by joschi99** [comment]
Found on /var/opt/rh/rh-php71/log/php-fpm/centreon-error.log error:

```
[20-Mar-2019 14:37:59 Europe/Rome] PHP Notice:  Undefined offset: 1 in /usr/share/centreon/www/index.php on line 112
[20-Mar-2019 14:38:06 Europe/Rome] PHP Notice:  Undefined offset: 1 in /usr/share/centreon/www/index.php on line 112
[20-Mar-2019 14:39:17 Europe/Rome] PHP Notice:  Element 'contact_passwd' does not exist in /usr/share/centreon/www/lib/HTML/QuickForm/HTML_QuickFormCustom.php on line 168
[20-Mar-2019 14:39:17 Europe/Rome] PHP Notice:  Element 'contact_passwd' does not exist in /usr/share/centreon/www/lib/HTML/QuickForm/HTML_QuickFormCustom.php on line 168
[20-Mar-2019 14:40:15 Europe/Rome] PHP Notice:  Element 'contact_passwd' does not exist in /usr/share/centreon/www/lib/HTML/QuickForm/HTML_QuickFormCustom.php on line 168
```

**Comment by joschi99** [comment]
With LDAP enabled also deploy to Poller is very slow, php-fpm is consuming a lot of CPU. If I disable LDAP authentication export to Poller is fast and php-fpm is not using CPU.

**Comment by adr-mo** [comment]
Hi,

This issue will be fixed in the 18.10.5.

Fixed by this PR https://github.com/centreon/centreon/pull/7278

Regards,

**Comment by hdep** [comment]
thank you to confirm that was identified, would have be nice to confirm that *before* fixing it.

**Comment by adr-mo** [comment]
Hi,

Sorry for the delay.

In fact this issue has been confirmed by other members of the community and internaly.

Regards,

**Comment by hdep** [comment]
Hi,

Still slow in 18.10.5 I just run some test right now.

**Comment by hdep** [comment]
hi @adr-mo any news about this ?

**Comment by adr-mo** [comment]
It is currently reworked by @sc979 

Hopefully it will be available in next release

Regards

**Comment by lolokai** [comment]
Any news about this feature ? Do you have a patch ?

**Comment by adr-mo** [comment]
Hi @lolokai @hdep 

We are currently reworking the LDAP synchronisation on Centreon.
The work made by @sc979 is currently being reviewed and tested by our QA team.

This should be available by the end of the month on 18.10.7 + 19.04.4 (the 18.10.6 and 19.04.3 being released this week)

Regards,

**Comment by wackou72** [comment]
@adr-mo  thanks for the feedback

**Comment by hdep** [comment]
I found a way to avoid this bug. SO I'll close this ticket because I don't even know if you really fix it on 18.10.7
